server/main.lua

```diff
@@ -204,16 +204,40 @@ local function format_int(number)
     return minus .. int:reverse():gsub("^,", "") .. fraction
 end
 
+-- Get all bank statements for the current player
+local function getBankStatements(cid)
+    local bankStatements = MySQL.Sync.fetchAll('SELECT * FROM bank_statements WHERE citizenid = ? ORDER BY record_id DESC LIMIT 30', { cid })
+    return bankStatements
+end
+
+-- Adds a bank statement to the database
+local function addBankStatement(cid, accountType, amountDeposited, amountWithdrawn, accountBalance, statementDescription)
+    local time = os.date("%Y-%m-%d %H:%M:%S")
+    MySQL.Async.insert('INSERT INTO `bank_statements` (`account`, `citizenid`, `deposited`, `withdraw`, `balance`, `date`, `type`) VALUES (?, ?, ?, ?, ?, ?, ?)', {
+        accountType,
+        cid,
+        amountDeposited,
+        amountWithdrawn,
+        accountBalance,
+        time,
+        statementDescription
+    }, function(_)
```

Comment:
This function can be removed completely, it is unnecessary if it doesn't execute any code

Reply:
fixed